docs/content/slides/pack-your-bags/index.md

```diff
@@ -1174,6 +1191,72 @@ Custom mixins are just easier to use, but aren't necessary.
 * Dominos 🍕
 * What else?
 
+---
+
+# Publishing Your Bundles
+
+Use  `porter publish` to share bundles:
+
+* Publishes both the invocation image and the CNAB bundle manifest to an OCI registry.
+* Uses Docker tags for both
+
+```yaml
+name: porter-azure-wordpress
+version: 0.1.0
+invocationImage: deislabs/porter-azure-wordpress:latest
+tag: deislabs/porter-azure-wordpress-bundle:latest
+```
+
+---
+
+## porter publish
+
+```console
+$ porter publish --help
+Publishes a bundle by pushing the invocation image and bundle to a registry.
+
+Usage:
+  porter publish [flags]
+
+Examples:
+  porter publish
+  porter publish --file myapp/porter.yaml
+  porter publish --insecure
+
+
+Flags:
+  -f, --file porter.yaml    Path to the Porter manifest. Defaults to porter.yaml in the current directory.
+  -h, --help                help for publish
+      --insecure-registry   Don't require TLS for the registry.
+```
+
+---
+
+# Try to publish
+
+* Modify the helloworld bundle again
+* Edit the porter.yaml:
+  * Change the `invocationImage` property to reflect your Docker Hub account
+  * Change `tag` property to reflect your Docker Hub account
+  * Change the message to something unique
+* Run `porter build`
+* Run `porter publish`
+
+Example (assuming your username is cnabaholic):
+
+* Change `deislabs/cool-image:latest` to `cnabaholic/cool-image:latest`
+
+---
+
+# Try to run your neighbor's bundle
```

Comment:
How safe is it to ask people to run arbitrary docker containers from people they don't know? I'd be leary of it though I do see what it is trying to show in the workshop.

Reply:
Probably a good point. I guess we could have them just change to another directory and run it from the tag they just pushed or remove this step entirely.